web/src/app.tsx

```diff
@@ -66,7 +43,50 @@ export default function App() {
             <Suspense fallback={<Loading />}>
               <NotificationBanner />
               <StyledRouter>
-                <AppRoutes />
+                <Routes>
+                  <Route path="/" element={<RootPage />} />
+                  <Route path="/login" element={<LoginPage />} />
+                  <Route path="/signup" element={<SignupPage />} />
+                  <Route path="/password-reset" element={<PasswordResetPage />} />
+                  <Route path="/dashboard/:workspaceId" element={<Dashboard />} />
+                  {/* classic routes - start */}
+                  <Route path="/edit/:sceneId" element={<EarthEditor />} />
+                  <Route path="/edit/:sceneId/preview" element={<Preview />} />
+                  <Route path="/plugin-editor" element={<PluginEditor />} />
+                  {/* classic routes - end */}
+                  <Route path="/scene/:sceneId" element={<BetaEditor />}>
+                    <Route path="story" element={<BetaEditor />} />
+                    <Route path="widgets" element={<BetaEditor />} />
+                    <Route path="publish" element={<BetaEditor />} />
+                  </Route>
+                  <Route path="settings" element={<Navigate to="/settings/account" />} />
+                  <Route path="/settings/account" element={<AccountSettings />} />
+                  <Route path="/settings/workspaces" element={<WorkspaceList />} />
+                  <Route path="/settings/workspaces/:workspaceId" element={<WorkspaceSettings />} />
+                  <Route
+                    path="/settings/workspaces/:workspaceId/projects"
+                    element={<SettingsProjectList />}
+                  />
+                  <Route
+                    path="/settings/workspaces/:workspaceId/asset"
+                    element={<AssetSettings />}
+                  />
+                  <Route path="/settings/projects/:projectId" element={<ProjectSettings />} />
+                  <Route path="/settings/projects/:projectId/public" element={<PublicSettings />} />
+                  <Route
+                    path="/settings/projects/:projectId/dataset"
+                    element={<DatasetSettings />}
+                  />
+                  <Route
+                    path="/settings/projects/:projectId/plugins"
+                    element={<PluginSettings />}
+                  />
```

Comment:
Grouping routes like `/scene` might be more readable.

Reply:
That is true. I'll make another PR refactoring current routes just in case switching to nested routing causes unexpected issues  👍🏻